plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -56,6 +56,7 @@
             - metric alert
             - service check
             - event alert
+            - event-v2 alert
```

Comment:
You also want to add
```
          - The type C(event-v2) was added in community.general 4.8.0.
```
in the `description` above.

Reply:
done